docs/source/utils/notebook.py

```diff
@@ -14,6 +14,13 @@ def inner(example_text: str):
 
 # %%
 !python3 -m pip install -q dff[examples]
+# Installs dff with dependencies for running examples
+# To install the minimal version of dff, use `pip install dff`
+# To install other options of dff, use `pip install dff[OPTION_NAME1,OPTION_NAME2]`
+# where OPTION_NAME can be one of the options from EXTRA_DEPENDENCIES.
+# e.g `pip install dff[ydb, mysql]` installs dff with dependencies for using Yandex Database and MySQL
+# EXTRA_DEPENDENCIES can be found in https://github.com/deeppavlov/dialog_flow_framework/blob/dev/setup.py
```

Comment:
It's a large file (more than 100 lines), so we should include line number:
```suggestion
# EXTRA_DEPENDENCIES can be found in https://github.com/deeppavlov/dialog_flow_framework/blob/dev/setup.py#L128
```

We could automate this process using this (just add this at the top of the file):
```
def get_extra_deps_line_number():
    with open("setup.py", "r", encoding="utf-8") as setup:
        return setup.readlines().index("EXTRA_DEPENDENCIES = {\n") + 1
```
And then
```suggestion
# EXTRA_DEPENDENCIES can be found in https://github.com/deeppavlov/dialog_flow_framework/blob/dev/setup.py#L{get_extra_deps_line_number()}
```